### DISCUSSION TITLE
[FEATURE]SOLO STUFF IDEAS

### DISCUSSION THREAD

**Discussion by rudeboey** [azerothcore/azerothcore-wotlk#3883] [Ideas]
HI 1st of all am new here.
i play azerothcore for sometime now and i have some ideas for solo play
(if any one can give me a link or a guide how to do this i will be gratefull)

multiplayer:is there any way except hamachi to a friend to conect at my server?
gameplay:as a solo player i want to now how to make reputation grow faster casue grinding 30 days for a mount its not so good and if is there any way to make my char exalted 2 all factions in a snap
or even if i can be friendly to opposite factions(like beign an orc and can visit stormwind and talk 2 npc's)

and if is there any way to make my custom npc like mount vendor or epic vendor etc(am new 2 this so any program suggestions would be awesome)

ty for the help

**Comment by Si1ker** [comment]
@rudeboey Hello and welcome here !
Remember to join the AzerothCore discord at https://discord.gg/gkt4y2x
(cc @FrancescoBorzi )

- `multiplayer: Is there any way except hamachi to a friend to conect at my server?`
You could try https://www.youtube.com/watch?v=E25-S6fGw0o


- `gameplay: As a solo player i want to now how to make reputation grow faster casue grinding 30 days for a mount its not so good`
In your build folder->()release/debug/relwithDebInfo)->worldserver.conf
You can change this line to 3, is not recommended to go beyond too high number as they would result in -numbers and that would make you hostile with the faction if you surpass the limit of reputation, this is a multiplayer for reputation.
https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/server/worldserver/worldserver.conf.dist#L2482


- `if is there any way to make my char exalted 2 all factions in a snap`
- `or even if i can be friendly to opposite factions(like beign an orc and can visit stormwind and talk 2 npc's)`
You could try to find the repu ID of stormwind,darnasus,forjaz,etc etc (alliance factions)
and do.. `.mod repu [ID of the reputation] 10000`
(to know the repu ID of a faction just use in google: wowhead kirin tor reputation, first result should be wowhead, at the end of the link is the reputation ID, like.... https://www.wowhead.com/faction=**72**/stormwind )

**Comment by rudeboey** [comment]
@Si1ker  
Ty dude i made some changes like 3xp default.1,3 move speed ,10 rep etc .drop rate 5
its more like diablo now and its awesome

btw any idea how to make my own vendor or how to merge classes?
